equinix/data_source_fabric_cloud_router_acc_test.go

```diff
@@ -0,0 +1,78 @@
+	"testing"
+)
+
+func TestAccDataSourceFabricCloudRouter_PFCR(t *testing.T) {
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ExternalProviders: testExternalProviders,
+		Providers:         testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: ConfigCreateCloudRouterResource(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "name", "Test_FCR"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "type", "XF_ROUTER"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "notifications.0.type", "ALL"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "notifications.0.emails.0", "test@equinix.com"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "order.0.purchase_order_number", "1-323292"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "location.0.metro_code", "SV"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "package.0.code", "LAB"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "project.0.project_id"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "account.0.account_number"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "href"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "state", "PROVISIONED"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "equinix_asn"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "bgp_ipv4_routes_count", "0"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "bgp_ipv6_routes_count", "0"),
+					resource.TestCheckResourceAttr("data.equinix_fabric_cloud_router.example", "connections_count", "0"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.created_by"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.created_by_full_name"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.created_by_email"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.created_date_time"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.updated_date_time"),
+					resource.TestCheckResourceAttrSet("data.equinix_fabric_cloud_router.example", "change_log.0.deleted_date_time"),
+				),
+				ExpectNonEmptyPlan: false,
+			},
+		},
+	})
+}
+
+func ConfigCreateCloudRouterResource() string {
+	return fmt.Sprintf(`
+		resource "equinix_fabric_cloud_router" "example" {
+		name = "Test_FCR"
```

Comment:
The resource names need to be tagged with the account they are apart of so that we can sweep them if necessary. Use PNFV or PFCR as the suffix. In this case, it would be PFCR.

Reply:
This still needs to have the PFCR suffix. Should be `Test_PFCR`